_providers/old/tiscali.it.md

```diff
@@ -1,9 +1,7 @@
 ---
+domains: tiscali.it
+last_checked: 2019-12
 name: Tiscali
+status: OK
 website: https://tiscali.it
-domains: tiscali.it
-credentials: emailPass
-status:
- state: OK
- date: 2019-12
 ---
```

Comment:
Why did you change the file in `old/` here? I guess this was meant to be moved to the parent directory?

Reply:
i think it can just be deleted, it was already updated in #72

Reply:
Oh, I missed that. Thanks!